test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -131,9 +131,9 @@ private enum RemoveType
                                    state -> ClusterUtils.waitForCMSToQuiesce(state.cluster, state.cmsGroup));
     }
 
-    private Command<State<S>, Void, ?> stopInstance(int toRemove)
+    private Command<State<S>, Void, ?> stopInstance(int toRemove, String why)
```

Comment:
this was a logging bug... `nodetool removenode` also calls this method, but we would log that it was for `assassinate`!